websites/G/Google Earth/presence.ts

```diff
@@ -0,0 +1,155 @@
+const presence = new Presence({
+		clientId: "1034133591938060298",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/bsN9g4H.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname } = window.location;
+
+	if (pathname === "/web/") presenceData.details = "Loading";
+	else {
+		const appRoot = document.querySelector("earth-app").shadowRoot,
+			knowledgeCard = appRoot.querySelector("earth-knowledge-card"),
+			drawerContainer = appRoot.querySelector("paper-drawer-panel"),
+			drawerRoot = drawerContainer.querySelector("earth-drawer").shadowRoot,
+			topToolbarRoot = appRoot.querySelector("earth-top-toolbar").shadowRoot,
+			imageLightBox = document.querySelector<HTMLElement>(
+				"earth-image-lightbox"
+			),
+			voyager = appRoot.querySelector<HTMLElement>("earth-voyager"),
+			measurementToolRoot =
+				drawerContainer.querySelector("earth-measure-tool").shadowRoot,
+			fullscreenBalloon = appRoot.querySelector<HTMLElement>(
+				"earth-balloon-fullscreen"
+			);
+		if (
+			pathname.startsWith("/web/search/") &&
+			appRoot
+				.querySelector("earth-toolbar")
+				.shadowRoot.querySelector("#search")
+				.getAttribute("active") === ""
```

Comment:
Empty strings are falsey which is probably best to check for instead of comparing it to an empty string